packages/common/src/components/utils/Accordion.tsx

```diff
@@ -36,9 +46,25 @@ const Accordion: React.FC<Props> = ({ title, description, additionalStyles, disp
     item.classList.toggle('ic-expandable--expanded')
   }
 
-  return (
-    <div className="ic-expandable" ref={expandableRef}>
-      <h4 className={`iu-fs-300 iu-color-black ${additionalStyles}`}>
+  const getHeader = () => {
+    if (!hasHeader) {
+      return (
+        <>
```

Comment:
Are these fragments needed here? (The empty <>). Just asking since there's just one line between them (same on line 59). Perhaps I'm missing something though